core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -110,6 +110,7 @@ public void init() throws StoreInitializationException {
       throw new StoreInitializationException(
           "Illegal state while initializing store. Store was already initialized");
     }
+    localStore.init();
```

Comment:
Why is the init required here?

Reply:
Currently `InMemoryCache.init` is a noop, but at one point I was trying something out where I had some code in `InMemoryCache.init` and was surprised that it was not called.  This seems like the natural place to call it.